sxs/simulations/analyze.py

```diff
@@ -154,17 +156,17 @@ def analyze_simulation(
     sim = load(sim_name)
 
     # Lev analysis
-    if analyze_levs:
+    if analyze_levs and not len(sim.lev_numbers) < 2:
         for i, lev in enumerate(sim.lev_numbers[1:][::-1]):
```

Comment:
You still have some `lev - 1`s hanging around in this block.  I suggest just iterating over pairs:
```suggestion
        for i, (low_lev, high_lev) in enumerate(zip(sim.lev_numbers[:-1], sim.lev_numbers[1:])):
```
In this case, your `i == 0` condition will need to change to `i == len(sim.lev_numbers)-2`.